profiles/ding2/modules/opensearch/includes/opensearch.search.inc

```diff
@@ -691,19 +676,14 @@ function opensearch_search_autocomplete($string) {
  *   Search suggestion results.
  */
 function opensearch_search_autocomplete_suggestions(array $query) {
-  $items = array();
-  $rest_url = variable_get('opensearch_search_autocomplete_suggestion_url', '');
-  if (empty($rest_url)) {
-    watchdog(
-      'opensearch_search_autocomplete',
-      "Autocomplete error: Empty opensearch autocomplete url",
-      array(),
-      $severity = WATCHDOG_ERROR
-    );
-    return $items;
-  }
+  // Hack to use old suggestions service.
+  $query += variable_get('ting_search_autocomplete_settings', array());
+  $query['query'] = $query['q'];
+  $query['maxSuggestions'] = $query['rows'];
+
+  $method = variable_get('ting_search_autocomplete_method', 'facets');
 
-  $url = url($rest_url, array('query' => $query));
+  $url = url('http://opensuggestion.addi.dk/b3.5_2.0/rest/'.$method, array('query' => $query));
```

Comment:
Should use the URL set in the admin and store in: `variable_get('opensearch_search_autocomplete_suggestion_url', '');`

Reply:
Which is set to `http://opensuggestion.addi.dk/b3.5_2.0`

Reply:
Well, this is a hack and it may make sense to hardcode the url …

Reply:
Then the hack should remove the option to set the URL in the admin UI. It don't make sens to have an UI to set the URL which have not effect.